src/main/java/Duke.java

```diff
@@ -1,10 +1,389 @@
+import java.util.ArrayList;
+import java.util.Objects;
+import java.util.Scanner;
+
 public class Duke {
```

Comment:
Different Classes can be placed in a separate file for better readability?
e.g Creating Task.java to store the Class `Task`